src/tapis-hooks/streams/measurements/infiniteQuery.ts

```diff
@@ -0,0 +1,102 @@
+import { Streams } from '@tapis/tapis-typescript';
+
+////////////////////////////
+//custom infiniteQuery ops//
+////////////////////////////
+
+export const tapisNextPageParam = (
+  lastPage: Streams.RespListMeasurements,
+  allPages: Streams.RespListMeasurements[],
+  params: { limit?: number }
+) => {
+  const measurements: Streams.Measurements | null = lastPage.result ?? null;
+  //get number of results
+  let results = 0;
+  if (measurements) {
+    //deconstruct to standard props and props referencing streams variables
+    const { instrument, site, measurements_in_file, ...variableProps } =
+      measurements;
+    for (let variable in variableProps) {
+      //get variable measurements
+      let variableMeasurements: { [datetime: string]: number } =
+        measurements[variable];
+      results += Object.keys(variableMeasurements).length;
```

Comment:
The [code below](https://github.com/tapis-project/tapis-ui/pull/175/files#diff-6fe98e47ee9136c4d0bcd43a974943cee7a83ab658516901f15b4ea00fd5a151R98) seems to indicate that `measurements_in_file` correlates to determining pagination size. Should that be used instead of manually computing result size?

Reply:
I think these are unrelated, I am not sure though. If I remember correctly measurements_in_file is a representation of the number of unique datetimes in the object. I will ask for clarification.